.travis/script_test.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+set -ev
+
+vendor/bin/phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
```

Comment:
I'd add this as custom composer script, so it can easily be executed on the command line.